### PR TITLE
15 review m1 projects

### LINKED ISSUE #15: Review M1 Projects
Go through all 5 [M1 projects](https://github.com/turingschool/curriculum-site/tree/main/module1/projects) and ensure that they are good to go for next inning. We don't anticipate any major changes, but review the time line of the lessons and ensure that students will be set up for success for each project. 

You may also choose to add an extension or something that has then write some JS code. 

Once you've deemed each project as "good-to-go," please go to [`module1/projects/index.md`](https://github.com/turingschool/curriculum-site/blob/main/module1/projects/index.md) and comment those projects IN so they appear on the [curriculum site](https://curriculum.turing.edu/module1/projects/).

### PR DESCRIPTION
**Pull request by juliet-e**
# Curriculum PR Template

### Does this PR close any issues?
*This PR closes #15  *

### Description
This PR:
- copies the projects over from the BE curriculum that we can use in 2407: Flashcards, The DMV, Battleship, and Futbol PD version
- adds alternative projects
- - War or Peace (can be swapped for Flashcards every other inning to give repeaters a new project)
- - Connect Four (more unstructured requirements for more advanced students to opt into for pair project)
- Adds an extension to the DMV project to write some JS objects using this repo: https://github.com/turingschool-examples/the_dmv_extension_js

### What questions do you have/what do you want feedback on? (optional)
- I would love for someone to actually DO the js extension based on the tests I wrote to make sure the tests are okay
- Do we need to add more JS extensions? I stopped after this one just for time's sake as this seems less urgent
- double check links/pages all work and route properly

### CODE REVIEW

**Comment by hfaerber** [comment]
I haven't reviewed this PR yet but I DID review and push up small edits to the JS exercise repo.  I think that one repo is enough for now and we can add to it in future iterations.  Nice job Juliet!

File: module1/projects/index.md [hunk 1]
```diff
@@ -6,10 +6,12 @@ title: Module 1 - Projects
...
-<!-- - Week 1-2 (Solo): [Flash Cards](./flashcards/) -->
-<!-- - Week 2-3 (Solo): [DMV](./dmv/) -->
-<!-- - Week 3-4 (Paired): [Battleship](./battleship/) -->
-<!-- - Week 5-6 (Group Final): [Futbol](./futbol/) -->
+
+- Week 1 (Ungraded): [Credit Check](./credit_check.markdown)
+<!-- Alternate between Flash Cards and War or Peace for repeaters -->
+- Week 1-2 (Solo): [Flash Cards](./flashcards/)
+- Week 2-3 (Solo): [DMV](./dmv/)
+<!-- Option to add more advanced option with Connect Four as other pair project -->
```
Comment (hfaerber) [inline module1/projects/index.md:14]: Do we want to add a link here for War or Peace and Connect Four but leave them commented out so they're easy to set up the following inning?

**Review by hfaerber — APPROVED** [review]
This looks great.  Added one comment wondering if commented out links for the alternate projects would be a good idea?

I've also reviewed the JS exercise repo and pushed up changes to a PR for that repo.   This looks ready to merge to me!

File: module1/projects/index.md (hunk 1 above)
Comment (juliet-e) [reply module1/projects/index.md:14]: Oh great call. I can do that and push!

**Pull request merged**